composer.json

```diff
@@ -23,6 +22,9 @@
     "support": {
         "issues": "https://github.com/phptek/silverstripe-jsontext/issues"
     },
+    "suggest": {
+      "silverstripe/cms": "Use the SendNotificationJob to queue and send notifications."
```

Comment:
Need to change this text to read something akin to "Use silverstripe/cms package if you want to write to JSON-backed DB fields via the CMS module's UI"